modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectValidationRuleResourceImpl.java

```diff
@@ -261,6 +262,28 @@ protected void preparePatch(
 					getObjectValidationRuleSettings()));
 	}
 
+	private com.liferay.object.model.ObjectValidationRuleSetting
+			_setObjectValidationRuleSettingProperties(
+				com.liferay.object.model.ObjectValidationRuleSetting
+					serviceBuilderObjectValidationRuleSetting,
```

Comment:
Hey @mateussgn , I'm just wondering if we should order this variable here, maybe as it starts with s, it should come after objectDefinitionId

Reply:
Sure, you are right!